support/qa/phpstan.neon

```diff
@@ -5,10 +5,4 @@ parameters:
     level: 7
     paths:
         - ../../src
-    ignoreErrors:
-        -
-            message: '#Method [a-zA-Z0-9\\_]+::[a-zA-Z0-9]+\(\) has parameter \$context#'
-            path: '**/Infrastructure/Serializer/*Normalizer.php'
-        -
-            message: '#Method [a-zA-Z0-9\\_]+::[a-zA-Z0-9]+\(\) return type has no value type specified in iterable type array#'
-            path: '**/Infrastructure/Serializer/*Normalizer.php'
+    checkMissingIterableValueType: false
```

Comment:
Please consider adding the reason for disabling this test as a comment to this line.